### ISSUE TITLE
RefreshFirebaseTokens error

### ISSUE THREAD

**Issue by Stampa-Stampa** [PrestaShopCorp/ps_accounts#421]
One of my Prestahop installations , having version 1.7.8.11 2 days ago stopped working...

In Front office (chrome browser) shows HTTP ERROR 500

Backoffice (chrome browser) shows the following error

"Oops... looks like an unexpected error occurred

Attempted to load trait "RefreshFirebaseTokens" from namespace "PrestaShop\Module\PsAccounts\Account\Session". Did you forget a "use" statement for "PrestaShop\Module\PsAccounts\Account\Session\RefreshFirebaseTokens"?

[Symfony\Component\Debug\Exception\ClassNotFoundException 0]
In server log i get the following errors

2024-07-18 21:53:09 | Warning | 52.143.70.100 |   | mod_fcgid: stderr: PHP Fatal error: Trait 'PrestaShop\Module\PsAccounts\Account\Session\RefreshFirebaseTokens' not found in /var/www/vhosts/buyclick.gr/stampa-stampa.com/modules/ps_accounts/src/Account/Session/ShopSession.php on line 35

I presume this happened due to changes in PS Modules (PrestaShop EventBus /account) cause my other installations at the same time had updates that were done while in this installation when i tried to connect to do the updates i could

Expected behavior
Basically login to back office and frontoffice working on all browsers as before 16/7/24

Steps to reproduce
Go to https://stampa-stampa.com/
PrestaShop version(s) where the bug happened
1.7.8.11

PHP version(s) where the bug happened
7.4.33 Fastcgi Apache

If your bug is related to a module, specify its name and its version
No response

Your company or customer's name goes here (if applicable).
Stampa-Stampa.com

**Comment by Stampa-Stampa** [comment]
I am back ...
i found out the following ..
The file ShopSession.php is looked for in the wrong url...
This is not normal as my installation has the following stracture
Stampa-Stampa.com/modules/ps_account/src/Account/Session/ShopSession.php

I checked both of my other eshop that have same php and prestahop versions (even theme) and in the stracture there is no account folder after ps_account...

I copied the ps_account folder and after i renamed the initial ps_account folder i replaced it with the one from my other installation ... and it worked...

I will need to check if all works fine like cart ets... but this is not normal....

**Comment by hschoenenberger** [comment]
Hello @Stampa-Stampa thanks for reporting your issue,
this is more likely to be an upgrade issue than a module issue IMHO.

Could you provide us with the version of the ps_accounts module that caused the issue ?

Also did you try to reset the module or clear the shop cache ?

Thanks
